src/shims/env.rs

```diff
@@ -42,7 +44,7 @@ impl<'tcx> EnvVars<'tcx> {
         ecx.update_environ()
     }
 
-    pub(super) fn values(&self) -> InterpResult<'tcx, Values<'_, OsString, Pointer<Tag>>> {
+    fn values(&self) -> InterpResult<'tcx, Values<'_, OsString, Pointer<Tag>>> {
         Ok(self.map.values())
     }
```

Comment:
I think you can just remove this method. Also, why does it return `InterpResult` when it is infallible?